README.md

```diff
@@ -1 +1,322 @@
+    When there is a pagination, the minimum limit is set to `20`.
+  - `fingerprint` The fingerprint of the last asset returned by the previous page.
+  - `order_by` Pre sorts the results during the query.
+    - Example:
+      `order_by=total_supply,asc` (starts from the rarest token)
+      `order_by=start_time,desc` (starts from the most recent ICO)
+  - `only_confirmed` Shows only the situation at latest confirmed block.
+    `true` | `false` default `false`
+
+### Blocks
+
+#### 1. Returns Events By Block Identifier
+
+- _GET_ https://api.trongrid.io/v1/blocks/:identifier/events
+- Usage:
+  - Returns all the events in the specified block. Depending on the data, the entire block can be confirmed or unconfirmed.
+- Params:
+  - `identifier` It can be either latest, a block number or a block id.
+
+### Contracts
+
+#### 1. Get Events By Contract Address
+
+- _GET_ https://api.trongrid.io/v1/contracts/:address/events
+- Usage:
+  - Returns the events emitted by a smart contract.
+- Params:
+  - `address` The address of the deployed contract.
+- Options:
+  - `only_confirmed` Shows only confirmed.
+    `true` | `false` default `false`
+  - `only_unconfirmed` Shows only unconfirmed.
+    `true` | `false` default `false`
+  - `event_name` The name of the event
+  - `block_number` The block number for which the events are required
+  - `min_block_timestamp` The minimum block timestamp default `0`
+    Alias: `min_timestamp`
+  - `max_block_timestamp` The maximum block timestamp default `now`
+    Alias: `max_timestamp`
+  - `limit` For pagination. Limit 20
+  - `fingerprint` The fingerprint of last event retrieved in the page
+  - `order_by` Sort the events.
+    - Accepted values:
+      `block_timestamp,asc` (alias: `timestamp,asc`)
+      `block_timestamp,desc` (default)
+
+#### 2. Get Transactions By Contract Address
+
+- _GET_ https://api.trongrid.io/v1/contracts/:address/transactions
+- Usage:
+  - Returns the transactions related a smart contract.
+- Params:
+  - `address` The address of the deployed contract.
+- Options:
+  - `only_confirmed` Shows only confirmed.
+    `true` | `false` default `false`
+  - `only_unconfirmed` Shows only unconfirmed.
+    `true` | `false` default `false`
+  - `min_block_timestamp` The minimum block timestamp default `0`
+    Alias: `min_timestamp`
+  - `max_block_timestamp` The maximum block timestamp default `now`
+    Alias: `max_timestamp`
+  - `limit` For pagination. Limit `20`
+  - `fingerprint` The fingerprint of last event retrieved in the page
+  - `order_by` Sort the events.
+    - Accepted values:
+      `block_timestamp,asc` (alias: `timestamp,asc`)
+      `block_timestamp,desc` (default)
+
+### Transactions
+
+#### 1. Get Events By Transaction ID
+
+- _GET_ https://api.trongrid.io/v1/transactions/:id/events
+- Usage:
+  - Returns the events emitted by a transaction.
+- Params:
+  - `id` The id of the transaction
+
+## The Following are Four Methods for Polling:
```

Comment:
Here we should refer to the legacy version (which will continue to be supported for retro-compatibility) linking to the README at
https://github.com/tronprotocol/tron-grid/tree/legacy

Reply:
It is linked at the top, I can add it at this section as well.